### PR TITLE
KH-480: Added release notes for version 1.6.1

### PR DESCRIPTION
**Pull request by kdaud**
Ticket → [KH-480](https://mekomsolutions.atlassian.net/browse/KH-456)

[KH-480]: https://mekomsolutions.atlassian.net/browse/KH-480?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
 
 ## Release Notes
 
+### Version 1.6.1
+
+- Added Disbursement report.
```
Comment (Ruhanga) [inline README.md:33]:
```suggestion
- Added Disbursement report improvements.
```

**Review by Ruhanga — CHANGES_REQUESTED** [review]

**Review by Ruhanga — COMMENTED** [review]
I think there's something more to do with i18n of drug related metadata?
cc @kazlaw

**Comment by kazlaw** [comment]
> I think there's something more to do with i18n of drug related metadata? cc @kazlaw

Yes actually but its more of i18n to Khmer

**Review by kazlaw — APPROVED** [review]
Looks okay to me @Ruhanga , @kdaud

**Review by Ruhanga — APPROVED** [review]

**Comment by kazlaw** [comment]
Cc: @rbuisson

File: README.md [hunk 2]
```diff
@@ -28,6 +28,13 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
 
 ## Release Notes
 
+### Version 1.6.1
+
+- Updated Common Reports OpenMRS module to version 1.4.1 containing Disbursement report improvements.
+- Added 'Unknown' option under HPV virus test in Cervical Cancer Screening form.
+- Translated the drug order form to Khmer.
```
Comment (rbuisson) [inline README.md:35]: Translated most of the drug order basket to Khmer.

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Review by rbuisson — APPROVED** [review]

**Pull request merged**